src/backend/web/graphql/index.js

```diff
@@ -35,6 +36,7 @@ module.exports.typeDefs = graphql`
     url: String
     guid: String
     feed: Feed!
+    user: String
```

Comment:
Same here, I don't think we need this here. We have a resolver that gives you a post and the feed that _owns_ the post, so you'd have instant access to the user (since it'd be stored in the feed)

Reply:
Fixed.